lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -7,33 +7,10 @@
 
   .profile-sidebar {
     background: transparent;
-    @include float(left);
-    @include margin-right(flex-gutter());
```

Comment:
@frrrances This would make the `main` content down-wards. There is already enough margin-left. I think we can skip this.